public/app/features/explore/TraceView/components/TracePageHeader/Actions/TracePageActions.tsx

```diff
@@ -16,7 +16,8 @@ export const getStyles = (theme: GrafanaTheme2) => {
       label: TracePageActions;
       display: flex;
       gap: 4px;
-      margin-top: 2px;
+      margin-top: 4px;
+      height: 32px;
```

Comment:
I've set up height of action items to 32px because that will be the default height of PanelChrome's header. Maybe we can use theme's configuration to get these values and not have to hard code them in.